jako/distribute/DistributedScan.py

```diff
@@ -111,38 +111,53 @@ def __init__(self,
         else:
             TypeError('`config` must be dict or filename string.')
 
+        # write database name as same as experiment name
+        self.config_data['database']['DB_TABLE_NAME'] = experiment_name
+
+        if isinstance(config, str):
+            config_path = config
+
+        else:
+            config_path = 'config.json'
+
+        with open(config_path, 'w') as f:
+            json.dump(self.config_data, f, indent=2)
+
         if 'finished_scan_run' in self.config_data.keys():
             del self.config_data['finished_scan_run']
 
         # handles location for params,data and model
-        if not os.path.exists("tmp/"):
-            os.mkdir("tmp")
+        if not os.path.exists("/tmp/"):
+            os.mkdir("/tmp/")
 
-        self.dest_dir = "./tmp"
+        self.dest_dir = "/tmp/"
```

Comment:
I think we don't use this anywhere? If that's the case, better remove it.

Reply:
It is used as the destination directory in https://github.com/autonomio/jako/blob/1a5196179f7cfc43a6ecff10f67157c47c7f1cc0/jako/distribute/distribute_utils.py#L144